### PR TITLE
Advisor-2082: introduce application tour

### PR DESCRIPTION
**Pull request by mkholjuraev**
Application tour UI design is ready. The design includes a button with id='advisor_pendo_intro' that needs to binded with Pendo tour application. This will be done by  Margot Menestrot.  

image

### CODE REVIEW

**Comment by terezanovotna** [comment]
Hi @mkholjuraev I don't see the button in prod, can we add it out there? the button will be then linked to this tour which is ready to go https://app.pendo.io/s/5300167311360000/guides/LR8rgW5-9fEM86HaWd2dWjT6vPU?view=settings

**Comment by terezanovotna** [comment]
And as we do not have tours ready right now for the rest of the applications, let's cut the content under the button "Take Advisor tour"
Screen Shot 2022-01-20 at 14 27 22 and link the pendo tour to that button
.

File: src/PresentationalComponents/AppIntro/AppIntro.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    <div>{intl.formatMessage(messages.introExploreInsights)}</div>
+  );
+
+  return (
+    <Popover
+      aria-label="Advisor intro wizard"
+      position={PopoverPosition.bottomEnd}
+      hideOnOutsideClick={false}
+      isVisible={show}
+      shouldClose={shouldClose}
```
Comment (johnsonm325) [inline src/PresentationalComponents/AppIntro/AppIntro.js:33]: You can remove `shouldClose` and `shouldOpen` functions in this component and simply pass `setShow` to the `shouldClose` and `shouldOpen` properties like so:
```
shouldClose={() => setShow(false)}
shouldOpen={() => setShow(true)}
```

**Review by johnsonm325 — CHANGES_REQUESTED** [review]
Everything seems good. Here's a suggested change.

**Comment by mkholjuraev** [comment]
I do not like this asking a review after a rebase all the time :)

**Review by jiridostal — APPROVED** [review]

**Pull request merged**